pages/techgenius/tg-installation.md

```diff
@@ -55,7 +55,7 @@ Open your `Terminal`. We assume that [brew](http://brew.sh/) is already installe
 
 ## Ubuntu
 
-Note: You may [download this bash file](uploads/bashScripts/instructions.sh) to automate the installation process. Make sure that the bash file is in the directory where you would like to store the [ole--vagrant-vi](https://github.com/dogi/ole--vagrant-vi) repository. Go to terminal and use command `sudo sh instructions.sh`
+Note: You may [download this bash file](https://github.com/open-learning-exchange/open-learning-exchange.github.io/blob/master/pages/uploads/bashScripts/instructions.sh) to automate the installation process. The bash file can be downloaded by right clicking on the Raw button and clicking on "save link as" present in the top right corner.Make sure that the bash file is in the directory where you would like to store the [ole--vagrant-vi](https://github.com/dogi/ole--vagrant-vi) repository. Go to terminal and use command `sudo sh instructions.sh`
```

Comment:
We want to use relative links as much as possible in our MDwiki. 

Reference: https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues/515

---

Aside from that, I'm not sure if this installation script section is necessary. It's only 6 lines. Plus there's no indication that this script covers the rest of the page's content. I feel `macOS`, `Ubuntu`, and `macOS and Ubuntu ONLY` kinda cluttered together, they could be reorganized better.

Reply:
please use no github link but relative one for the first link ...